### PR TITLE
Gst plugins web imagebitmapsrc

### PR DESCRIPTION
**Pull request by cfoch**

### CODE REVIEW

File: gst.wasm/subprojects/gst-plugins-web/gst/imagebitmap/gstimagebitmapsrc.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/* GStreamer
+ * Copyright (C) <2002> David A. Schleef <ds@schleef.org>
```
Comment (ylatuya) [inline gst.wasm/subprojects/gst-plugins-web/gst/imagebitmap/gstimagebitmapsrc.h:2]: Copyright is wrong

File: gst.wasm/subprojects/gst-plugins-web/gst/imagebitmap/gstimagebitmapsrc.cpp [hunk 2]
```diff
@@ -0,0 +1,483 @@
+/* Copyright (C) Fluendo S.A. <support@fluendo.com> */
```
Comment (ylatuya) [inline gst.wasm/subprojects/gst-plugins-web/gst/imagebitmap/gstimagebitmapsrc.cpp:1]: License missing in headers.
Year missing in Copyright

**Comment by rgonzalezfluendo** [comment]
fix conflicting files

**Comment by rgonzalezfluendo** [comment]
You need to update https://github.com/fluendo/gst.wasm/blob/main/cerbero/recipes/gst.wasm-samples.recipe#L12

**Comment by cfoch** [comment]
@rgonzalezfluendo done

**Pull request merged**